bbk-requirements.md

```diff
@@ -0,0 +1,76 @@
+# Secure Payment Confirmation: Browser Based Key Requirements and Design Considerations
+
+Status: This is a **draft** document without consensus.
+
+Though this document we seek to build consensus around requirements and design considerations for adding a browser-based key pair (BBK) to the results of Secure Payment Confirmation to serve as a possession factor during authentication. For discussion, see [issue 271](https://github.com/w3c/secure-payment-confirmation/issues/271).
+
+## Motivation
+
+Secure Payment Confirmation provides a convenient "sign what you see" experience for a user to agree to the terms and conditions of a transaction, and where Web Authentication is used to generate cryptographic evidence of the user's agreement.
+
+The payments industry has indicated that SPC would further benefit from a device binding capability. As WebAuthn passkeys can now be synced, it can be argued that they no longer meet strict 2FA requirements (being no longer a signal of device possession), and so SPC (like WebAuthn) is reduced to a single factor (biometric or possession, depending on the authentication method used).
```

Comment:
> reduced to a single factor (biometric or possession, depending on the authentication method used).

In what case can SPC/WebAuthn still provide possession, given syncing? Was this meant to read "biometric or knowledge" ?

Reply:
Yep, that was a bug. I've fixed it in the merged version. Thank you!